include/rviz_visual_tools/rviz_visual_tools.h

```diff
@@ -594,6 +594,17 @@ class RvizVisualTools
                    const std::string &ns = "Path");
 
   /**
+   * \brief Display a marker of a series of connected colored lines
+   * \param path - a series of points to connect with lines
+   * \param colors - a series of colors
+   * \param radius - the thickness of the line
+   * \param ns - namespace of marker
+   * \return true on success
+   * \note path and colors vectors must be the same size
+   */  bool publishPath(const std::vector<Eigen::Vector3d> &path, const std::vector<colors> &colors, const double radius = 0.01,
+                   const std::string &ns = "Path");
```

Comment:
The new function header is [here](https://github.com/davetcoleman/rviz_visual_tools/blob/kinetic-devel/include/rviz_visual_tools/rviz_visual_tools.h#L647) and has been released into kinetic [here](https://github.com/ros/rosdistro/pull/12080)